SPECS/qemu-dp.spec

```diff
@@ -98,10 +98,16 @@ Patch87: make-blockdev-snapshot-use-same-node-name.patch
 Patch88: use_existing_io_context.patch
 Patch89: use_libaio_by_default.patch
 Patch90: Use_the_legacy_grant_copy_ioctl
+
+#XCP-ng patches
+
+Patch1000: qemu-dp-2.10.2-add-rbd-support.XCP-ng.patch
+
 BuildRequires: libaio-devel glib2-devel
 BuildRequires: libjpeg-devel libpng-devel pixman-devel libdrm-devel
 BuildRequires: xen-dom0-devel xen-libs-devel libusbx-devel
 BuildRequires: libseccomp-devel
+BuildRequires: librados-devel librbd-devel
```

Comment:
It's good practice to add one separate BuildRequires: line per build requirement. I know, they didn't follow that good practice themselves, but let's do it for our own additions.

Reply:
I can't find those packages in CentOS (and neither can yum-builddep) but there's a librados2-devel and a librados1-devel package.

Reply:
These packages become available after installation of  centos-release-ceph-{version} package which creates a new yum repo for Ceph. 
I will add  centos-release-ceph-{version} to BuildRequires:

Reply:
I don't think you can do that. You have no guarantee that the centos-release-ceph-{version} package is installed before the others, and we can't have our build process depend on an external repository if we want builds to be reproduceable. 
We'll have to import those dependencies in our repositories.